buble.rb

```diff
@@ -1 +1,39 @@
-puts "hello"
+def buble_sort(arr = [1, 0, 4, 3, 5, 1, 0, 1])
+  i = 0
+  a = 1
+  arrl = arr.length - 1
+  while a != 0
+    a = 0
+    i = 0
+    while i < arrl
+      x = arr[i]
+      y = arr[i + 1]
+      if x > y
+        arr[i] = y
+        arr[i + 1] = x
+        a += 1
+      end
+      i += 1
+    end
+  end
```

Comment:
What the other TSE meant is that you should remove only `puts`
Now lets add `arr` back